README.md

```diff
@@ -29,15 +29,9 @@
 [![Build python source distribution and wheels](https://github.com/apache/beam/workflows/Build%20python%20source%20distribution%20and%20wheels/badge.svg?branch=master&event=schedule)](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
```

Comment:
This looks better to me - while we're at it, can we add a go badge?

`[![Go Reference](https://pkg.go.dev/badge/github.com/apache/beam/sdks/go.svg)](https://pkg.go.dev/github.com/apache/beam/sdks/go)` renders [![Go Reference](https://pkg.go.dev/badge/github.com/apache/beam/sdks/go.svg)](https://pkg.go.dev/github.com/apache/beam/sdks/go)